docs/qa/method.md

```diff
@@ -0,0 +1,182 @@
+---
+order: 1
+title: Method
+---
+
+The (first iteration of the) QA process as described [here][releases] is applied to version v0.34.x
+in order to have a set of results acting as benchmarking baseline.
+This baseline is then compared with results obtained in later versions.
+
+Out of the testnet-based test cases described in [releases][releases] we are focusing on two of them:
+_200 Node Test_, and _Rotating Nodes Test_.
+
+[releases]: https://github.com/tendermint/tendermint/blob/v0.37.x/RELEASES.md#large-scale-testnets
+
+# Software Dependencies
+
+## Infrastructure Requirements to Run the Tests
+
+* An account at Digital Ocean (DO), with a high droplet limit (>202)
+* The machine to orchestrate the tests should have the following installed:
+    * A clone of the [testnet repository][testnet-repo]
+        * This repository contains all the scripts mentioned in the reminder of this section
+    * [Digital Ocean CLI][doctl]
+    * [Terraform CLI][Terraform]
+    * [Ansible CLI][Ansible]
+
+[testnet-repo]: https://github.com/interchainio/tendermint-testnet
+[Ansible]: https://docs.ansible.com/ansible/latest/index.html
+[Terraform]: https://www.terraform.io/docs
+[doctl]: https://docs.digitalocean.com/reference/doctl/how-to/install/
+
+## Requirements for Result Extraction
+
+* Matlab or Octave
+* Prometheus server installed
+* blockstore DB of one of the full nodes in the testnet
+* Prometheus DB
+
+# 200 Node Testnet
+
+## Running the test
+
+This section explains how the tests were carried out for reproducibility purposes.
+
+1. [If you haven't done it before]
```

Comment:
This is a very low level of detail, I'm wondering what utility you think there is from describing things in this much detail? Is this for (1) us/future runners of the benchmark or is it for (2) users of the software who want to understand how the benchmarks were produced? 

For (2) I think that this is much too low level and makes it a bit difficult to understand the basic things a user might want to know like "what was the power of the machines?" and "what was the network connectivity and configuration like?" etc that they may want to get at a glance.

Reply:
When I wrote it, I was rather thinking of (1). Basically improving the bus factor, in case Thane you and I were offline for a while and someone else in the team had to take over these activities.
I have to say that, without this level of detail, I imagine myself trying to do this for the first time (and not being able to Zoom you :-D) and I would be in for a hard time.

Reply:
Sure thing!

Reply:
Much clearer!